webui/react/.eslintignore

```diff
@@ -1,6 +1,7 @@
 src/serviceWorker.ts
 src/services/api-ts-sdk/
 src/vendor/notebook.js
```

Comment:
chore: since it moved, can we drop this line?

Reply:
done!